ui-packages/packages/form-displayer/src/envelope/components/FormDisplayer/FormDisplayer.tsx

```diff
@@ -105,7 +114,7 @@ export const FormDisplayer = React.forwardRef<
 
     return (
       <div {...componentOuiaProps(ouiaId, 'form-displayer', ouiaSafe)}>
-        {isEnvelopeConnectedToChannel && !isExecuting ? (
+        {canDisplayForm ? (
```

Comment:
```suggestion
        {canDisplayForm() ? (
```

@jomarko I think you missed this here :D